src/EnergyPlus/FluidProperties.hh

```diff
@@ -513,7 +513,9 @@ namespace FluidProperties {
         Int64 const Grid_Shift(28);
         assert(Grid_Shift == 64 - 12 - t_sh_precision_bits);
```

Comment:
While we are at this, you can change the first line to:
`constexpr std::uint64_t Grid_Shift  = 64 - 12 - t_sh_precision_bits;
`I think this optimization was put in place before constexpr was added to C++.